libs/net/dal/Services/ContentService.cs

```diff
@@ -223,8 +225,17 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWi
         if (!string.IsNullOrWhiteSpace(filter.Headline))
             filterQueries.Add(s => s.Wildcard(m => m.Field(p => p.Headline).Value($"*{filter.Headline.ToLower()}*")));
 
-        // if (!string.IsNullOrWhiteSpace(filter.Keyword))
-        //     filterQueries.Add(s => s.Terms(t => t.Field(p => p.)));
+        if (!string.IsNullOrWhiteSpace(filter.Keyword))
+            filterQueries.Add(s => s.MultiMatch(m => m
+                .Fields(f => f
+                    .Field(p => p.Headline)
+                    .Field(p => p.Body)
+                    .Field(p => p.Byline)
+                    .Field(p => p.Contributor)
+                )
+                .Query(filter.Keyword.ToLower())
```

Comment:
Not sure `ToLower()` makes sense here.  All the field values would have to be lowercase too.  Is Elasticsearch case sensitive?

Reply:
I noticed Cello also did the same in the above queries.  Please test.

Reply:
Will do